devtools/maven/src/main/java/io/quarkus/maven/BuildFileMojoBase.java

```diff
@@ -137,6 +131,39 @@ public void execute() throws MojoExecutionException {
         }
     }
 
+    private QuarkusPlatformDescriptor loadPlatformDescriptor(final MavenArtifactResolver mvn, final MessageWriter log,
+            Artifact descrArtifact) {
+        return QuarkusJsonPlatformDescriptorResolver.newInstance()
+                .setArtifactResolver(new BootstrapAppModelResolver(mvn))
+                .setMessageWriter(log)
+                .resolveFromJson(descrArtifact.getFile().toPath());
+    }
+
+    private Artifact resolveJsonOrNull(MavenArtifactResolver mvn, String bomGroupId, String bomArtifactId, String bomVersion) {
+        Artifact jsonArtifact = new DefaultArtifact(bomGroupId, bomArtifactId, null, "json", bomVersion);
+        try {
+            jsonArtifact = mvn.resolve(jsonArtifact).getArtifact();
+        } catch (Exception e) {
+            if (getLog().isDebugEnabled()) {
+                getLog().debug("Failed to resolve JSON descriptor as " + jsonArtifact);
```

Comment:
Add the exception stacktrace to the log statement

Reply:
Same as above.